lib/spack/spack/test/installer.py

```diff
@@ -102,7 +102,13 @@ def test_hms(sec, result):
 def test_install_msg():
```

Comment:
I think this should use monkeypatch to change the tty._debug level so that it can be called from `SpackCommand` without messing with the users debug setting.

Reply:
Will do.

Reply:
Changed.